File: game-server/src/main/java/org/forfun/mmorpg/game/cross/message/Rpc_G2C_FetchFightServerNodes.java
package org.forfun.mmorpg.game.cross.message;

import jforgame.socket.share.annotation.MessageMeta;
import jforgame.socket.share.message.Message;
import lombok.Getter;
import lombok.Setter;
import org.forfun.mmorpg.game.MessageSource;
import org.forfun.mmorpg.game.cross.constant.ConstantCross;

@Getter
@Setter
@MessageMeta(cmd = ConstantCross.CMD_G2C_FETCH_FIGHT_SERVER_NODES, source = MessageSource.FROM_SERVER_TO_SERVER)
public class Rpc_G2C_FetchFightServerNodes implements Message {

}

File: game-server/src/main/kotlin/org/forfun/mmorpg/game/cross/message/Rpc_G2C_FetchFightServerNodes.kt
package org.forfun.mmorpg.game.cross.message

import jforgame.socket.share.annotation.MessageMeta
import jforgame.socket.share.message.Message
import org.forfun.mmorpg.game.MessageSource
import org.forfun.mmorpg.game.cross.constant.ConstantCross

@MessageMeta(cmd = ConstantCross.CMD_G2C_FETCH_FIGHT_SERVER_NODES, source = MessageSource.FROM_SERVER_TO_SERVER)
class Rpc_G2C_FetchFightServerNodes : Message
